static_analyzer/incremental_orchestrator.py

```
@@ -85,11 +93,247 @@ def update_cfg_for_changed_files(
+    for file_path in changed_source_files:
+        call_sites = source_inspector.find_call_sites(file_path)
+        if not call_sites:
+            continue
+        queries = [(file_path, line, char) for line, char in call_sites]
+        try:
+            definition_results, _ = engine_client.send_definition_batch(queries)
+        except Exception:
+            logger.debug("Failed to resolve outbound definitions for %s", file_path, exc_info=True)
+            continue
+
+        for (line, char), definitions in zip(call_sites, definition_results):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)